package.json

```diff
@@ -80,51 +80,61 @@
       {
         "command": "cmake.viewLog",
         "title": "%cmake-tools.command.cmake.viewLog.title%",
+        "when": "cmake:enableFullFeatureSet",
         "category": "CMake"
       },
       {
         "command": "cmake.selectActiveFolder",
         "title": "%cmake-tools.command.cmake.selectActiveFolder.title%",
+        "when": "cmake:enableFullFeatureSet",
         "category": "CMake"
       },
       {
         "command": "cmake.editKits",
         "title": "%cmake-tools.command.cmake.editKits.title%",
+        "when": "cmake:enableFullFeatureSet",
         "category": "CMake"
       },
       {
         "command": "cmake.scanForKits",
         "title": "%cmake-tools.command.cmake.scanForKits.title%",
+        "when": "cmake:enableFullFeatureSet",
         "category": "CMake"
       },
       {
         "command": "cmake.selectKit",
         "title": "%cmake-tools.command.cmake.selectKit.title%",
+        "when": "cmake:enableFullFeatureSet",
         "category": "CMake"
       },
       {
         "command": "cmake.setVariant",
         "title": "%cmake-tools.command.cmake.setVariant.title%",
+        "when": "cmake:enableFullFeatureSet",
         "category": "CMake"
       },
       {
         "command": "cmake.setVariantAll",
         "title": "%cmake-tools.command.cmake.setVariantAll.title%",
+        "when": "cmake:enableFullFeatureSet",
         "category": "CMake"
       },
       {
         "command": "cmake.configure",
         "title": "%cmake-tools.command.cmake.configure.title%",
+        "when": "cmake:enableFullFeatureSet",
```

Comment:
I'm thinking that maybe this command can always stay active so that if a user creates a CMakeLists.txt file in a folder that doesn't have one, they can use this command as an entry point to enable the extension. I've been doing this when testing previous PR's, so I noticed when I wasn't able to do it with this one.

Reply:
Yes good idea. I realize now that all configure commands and 'quick start' make sense to remain active.